be/src/exprs/table_function/table_function_factory.h

```diff
@@ -9,10 +9,34 @@ namespace starrocks::vectorized {
 
 class TableFunctionFactory {
 public:
-    static TableFunctionPtr MakeUnnest();
 };
 
 extern const TableFunction* get_table_function(const std::string& name, const std::vector<PrimitiveType>& arg_type,
                                                const std::vector<PrimitiveType>& return_type,
                                                TFunctionBinaryType::type binary_type = TFunctionBinaryType::BUILTIN);
+
+// List of all table functions
+
+// Name: json_each
+// Usage: expands the outermost JSON object into a set of key/value pairs.
+// Signature: table(key, value) json_each(JsonColumn)
+class JsonEach final : public TableFunction {
```

Comment:
Why put the tablet function definition in factory class?

Reply:
Cause right now there's no place to declare all table functions, I think it's better to declare all table functions in this header than declare all of them in individual files ?

Reply:
Currently. Scala functions and aggregate functions are declared in individual files.  we have put all function signatures to xxx function factory，from xxx function factory，we could know which function we have，So I think which isn't necessary to define all functions in function factory.

Reply:
Fine.